pages/downloads.md

```diff
@@ -56,25 +55,25 @@ Be sure to verify your downloads by these [procedures][VERIFY_PROCEDURES] using
 
 ## Current Releases
 
-### 2.0.0-alpha-2 **preview**{: .label .label-primary}
+### 2.0.0 **latest**{: .label .label-primary }
 
-The 2.0.0-alpha-2 release of Apache Accumulo&reg; has been made available as a
-preview release for 2.0.0. This version is *not* suitable for production use,
-and should be used for feedback and testing purposes only. See the
-[release notes][REL_NOTES_20] for more details.
+The 2.0.0 release of Apache Accumulo&reg; is the latest release, containing
+the newest features, bug fixes, performance enhancements, and more.
+See the [release notes][REL_NOTES_20] for more details about this release.
 
 {: .table }
-| **Binary** | [accumulo-2.0.0-alpha-2-bin.tar.gz][BIN_20] | [ASC][ASC_BIN_20] | [SHA][SHA_BIN_20] |
-| **Source** | [accumulo-2.0.0-alpha-2-src.tar.gz][SRC_20] | [ASC][ASC_SRC_20] | [SHA][SHA_SRC_20] |
+| **Binary** | [accumulo-2.0.0-bin.tar.gz][BIN_20] | [ASC][ASC_BIN_20] | [SHA][SHA_BIN_20] |
+| **Source** | [accumulo-2.0.0-src.tar.gz][SRC_20] | [ASC][ASC_SRC_20] | [SHA][SHA_SRC_20] |
 
 #### 2.0 Documentation
 * [README][README_20]
 * [Online Documentation][MANUAL_20]
 * [Java API][JAVADOC_20]
 
-### 1.9.3 **latest**{: .label .label-primary }
```

Comment:
What does **maintenance** mean?

Reply:
Not sure, precisely. I was thinking it was a maintenance release for 1.9. Most importantly, it wasn't "latest". I changed it to "previous", which I think is only marginally better. I could remove the label entirely, but then there's not really a visual marker delineating one release from the next one on the page.